### PR TITLE
feat: Bill of lading number

### PR DESCRIPTION
**Pull request by gunnar2k**
Adds support for BOL number through public api

### CODE REVIEW

File: vic.api.v0.yaml [hunk 1]
```diff
@@ -2169,6 +2169,10 @@ components:
           description: Number that appears on the invoice.
           type: string
           maxLength: 255
+        bolNumber:
+          description: The bill of lading number.
+          type: string
+          nullable: true
```
Comment (warmwaffles) [inline vic.api.v0.yaml:2175]:
```suggestion
        bolNumber:
          description: The bill of lading number.
          type: string
          maxLength: 255
          nullable: true
```

File: vic.api.v0.yaml [hunk 2]
```diff
@@ -2728,6 +2732,10 @@ components:
           description: number that appears on the invoice
           type: string
           maxLength: 255
+        bolNumber:
+          description: The bill of lading number.
+          type: string
+          nullable: true
```
Comment (warmwaffles) [inline vic.api.v0.yaml:2738]:
```suggestion
        bolNumber:
          description: The bill of lading number.
          type: string
          maxLength: 255
          nullable: true
```

File: vic.api.v0.yaml [hunk 3]
```diff
@@ -2780,6 +2788,10 @@ components:
           type: string
           maxLength: 255
           nullable: true
+        bolNumber:
+          description: The bill of lading number.
+          type: string
+          nullable: true
```
Comment (warmwaffles) [inline vic.api.v0.yaml:2794]:
```suggestion
        bolNumber:
          description: The bill of lading number.
          type: string
          maxLength: 255
          nullable: true
```

**Review by warmwaffles — COMMENTED** [review]
Just need to add `maxLength`. Once this is merged, I'll chuck a line in the changelog, unless you want to.

**Comment by gunnar2k** [comment]
@warmwaffles Done! Let me know :pray:

**Review by warmwaffles — APPROVED** [review]

**Pull request merged**